pkg/cmd/target/set.go

```diff
@@ -169,3 +192,114 @@ var TargetSetCmd = &cobra.Command{
 		return nil
 	},
 }
+
+func handleTargetJSON(data []byte) error {
+	ctx := context.Background()
```

Comment:
![Screenshot 2024-12-10 at 17 06 33](https://github.com/user-attachments/assets/ae4e83e2-1081-4646-bb2f-37b7f1d8bfde)

You need to throw an error if the user defines properties that are not in the manifest.

Reply:
I have only been able to figure it out for local docker targets, does i also apply to other provider manifests to

Reply:
Of course.
If I were to set, `"SOMETHING INVALID": "invalid"` in the target config options, it needs to throw an error because `SOMETHING INVALID` is not part of the manifest

Reply:
Alright, i will need sometime to figure it out, what i have in mind currently is to follow this [approach](https://github.com/daytonaio/daytona/pull/1380/commits/b7435d0233690fd3c31dc50d35a0b4d82141a070) but we can't hardcode things right

Reply:
Take your time. No hardcoding, yes